src/Akeneo/Pim/Enrichment/Component/Category/Connector/ArrayConverter/FlatToStandard/Category.php

```diff
@@ -17,12 +19,18 @@ class Category implements ArrayConverterInterface
     /** @var FieldsRequirementChecker */
     protected $fieldChecker;
 
+    /** @var LocaleRepositoryInterface */
+    protected $localeRepository;
+
     /**
      * @param FieldsRequirementChecker $fieldChecker
      */
-    public function __construct(FieldsRequirementChecker $fieldChecker)
-    {
+    public function __construct(
+        FieldsRequirementChecker $fieldChecker,
+        LocaleRepositoryInterface $localeRepository
```

Comment:
to avoid BC break : 

```suggestion
        LocaleRepositoryInterface $localeRepository = null
```
and add some `if (null !== $this->localeRepository)` where it's needed

Reply:
I'm not very familiar with this BC break convention, I just saw it a couple of times, maybe you could ask some precision to your LD.

Reply:
@ValentinMumble any thought on that ?